File: app/src/main/java/za/co/duartbreedt/androidconceptsreference/NextActivity.java
package za.co.duartbreedt.androidconceptsreference;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import za.co.duartbreedt.androidconceptsreference.databinding.ActivityMainBinding;
import za.co.duartbreedt.androidconceptsreference.databinding.ActivityNextBinding;

public class NextActivity extends AppCompatActivity {

    private ActivityNextBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityNextBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }
}

File: app/src/main/java/za/co/duartbreedt/androidconceptsreference/NextActivity.kt
package za.co.duartbreedt.androidconceptsreference

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import za.co.duartbreedt.androidconceptsreference.databinding.ActivityNextBinding

class NextActivity : AppCompatActivity() {

    private lateinit var binding: ActivityNextBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityNextBinding.inflate(layoutInflater)

        setContentView(binding.root)
    }
}